### PR TITLE
[feat] Add cassandra example

### PR DESCRIPTION
**Pull request by lukaszewczak**
Hi @gergelyke ,

I finally made this PR with Cassandra example. Sorry for taking it so long.

### CODE REVIEW

File: example/cassandra/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  }
+}
+
+client.connect()
+  .then(() => {
+    console.log('db connected')
+
+    terminus(http.createServer(app), {
+      logger: console.log,
+      signal: 'SIGINT',
```
Comment (gergelyke) [inline example/cassandra/index.js]: is this on-purpose? `SIGTERM` should be used for production environments, shouldn't it?

Comment (lukaszewczak) [reply example/cassandra/index.js]: I'm on Windows where `SIGTERM` is not supported, the same situation is in all examples.

Comment (gergelyke) [reply example/cassandra/index.js]: Oh, okay, makes sense

**Pull request merged**